charts/eco_breakdown_material_footprint_overshoot/eco_breakdown_material_footprint_overshoot.js

```diff
@@ -0,0 +1,136 @@
+                }
+            });
+        } else {
+            console.error("failed to load data ", err)
+        }
+
+
+
+    });
+}
+
+//--------------------------------------- showChart
+function render_eco_breakdown_material_footprint_overshoot(canvasID) {
+
+    var chartID = "eco_breakdown_material_footprint_overshoot"
+    var options = {
+        chart: {
+            type: 'bar',
+            stacked: true,
+            height: '3500',
+            fontFamily: 'Open Sans',
+            toolbar: {
+                show: false,
+                tools: { zoom: false }
+            },
+        },
+        theme: {
+            palette: 'palette3',
+        },
+        responsive: [
+            {
+                breakpoint: 960,
+                options: {
+                    xaxis: { tickAmount: 1 }
+                }
+            },
+            {
+                breakpoint: 601,
+                options: {
+                    tooltip: {
+                        y: {
+                            title: {
+                                // Names are too long for mobile view
+                                // Alternatively, formatter could change seriesName to abbreviation
+                                formatter: (seriesName) => '',
+                            }
+                        },
+                    }
+                }
+
+            },
+        ],
+        plotOptions: {
+            bar: {
+                horizontal: true,
+            }
+        },
+        dataLabels: {
+            enabled: false
+        },
+        yaxis: {
+            //max: 3.8e11,
+            labels: { align: 'left'}
+        },
+        xaxis: {
+            tickAmount: 1,
+            position: 'top',
+            axisBorder: { show: false },
+            labels: { formatter: (val, index) => formatYAxisLabel(val, index, 0)+' Gt'},
+            axisTicks: { height: 0 }
+        },
+        colors: ['#775DD0', '#73c71c'],
+        grid: {
+            padding: {
+                top: 0,
+            }
+        },
+        tooltip: {
+            y: {
+                formatter: (val, index) => formatTooltipVal(val, index),
+            },
+            followCursor: true,
+            shared: false,
+        },
+    }
+
+
+    let sorted = window.chart_data["eco_breakdown_material_footprint_overshoot"].data.sort((a, b) => b['Cumulative-MF-overshoot'] - a['Cumulative-MF-overshoot']);
+    //sorted = sorted.filter(x => x.iso != 'GUY'); // EXCLUDING Guyana
+    let countries = [];
+    let series = [
+        { name: 'Gigatonnes (Gt)', data:[]},
+    ];
+    sorted.forEach(row => {
+        let overshoot = +row['Cumulative-MF-overshoot'];
+        if (overshoot > 0) {
+            series[0].data.push(overshoot);
+            countries.push(window.chart_data["eco_breakdown_material_footprint_overshoot"].countries[row['category']]);
+        }
+    });
+    console.log(series, sorted)
```

Comment:
Please remove all console logs

Reply:
The console.log in charts.php are behind a `if $debug == true` so they never get called. What i meant was, that we need to keep the log to the console as minimal as possible, so only error logs and nothing else :)